packages/jupyter-chat/src/model.ts

```diff
@@ -169,11 +169,12 @@ export interface IChatModel extends IDisposable {
 }
 
 /**
- * The default chat model implementation.
- * It is not able to send or update a message by itself, since it depends on the
- * chosen technology.
+ * An abstract implementation of IChatModel.
+ *
+ * The class inheriting from it must implement at least:
+ * - sendMessage(message: INewMessage)
  */
-export class ChatModel implements IChatModel {
+export abstract class ChatModel implements IChatModel {
```

Comment:
Can this also be renamed to `AbstractChatModel` for clarity?

Reply:
Thanks, I updated it.